src/lib/classes/services/localStorageService.ts

```diff
@@ -14,15 +15,16 @@ export class LocalStorageService {
         if (window.localStorage) {
             try {
                 const json = window.localStorage.getItem(key);
-
+                if (key !== "persisted data version" && json === null) {
```

Comment:
why do we run the migration if the key is not this one? We cant make sure the entire local storage is loaded at this point I think